CONTRIBUTING.md

```diff
@@ -56,6 +56,35 @@ This project follows the
 [Oracle Java code conventions](http://www.oracle.com/technetwork/java/codeconvtoc-136057.html)
 and the Scala conventions detailed above, preferring the latter.
 
+### Test
+There are two types of tests in this project. Unit tests that are written in Scala, and integration tests that are written in python. 
+We encourage writing integration tests if you have to choose between the two as that helps us ensure the plugin is working across different platforms. 
+
+#### Unit tests
+Unit-tests are located in com.nvidia:rapids-4-spark-tests_2.12. In order to run the unit-tests follow these steps
```

Comment:
I don't think listing a maven group and artifact ID is helpful here.  Why not point them to the test Maven project in this very repo?